nf_core/pipeline-template/.github/workflows/download_pipeline.yml

```diff
@@ -14,15 +14,10 @@ on:
   pull_request:
     types:
       - opened
-      - edited
       - synchronize
```

Comment:
Reading the docs, I see by default this is triggered by `opened`, `synchronize` or `reopened`. So we could completely remove the `types` here.